cli/src/main/java/com/github/ruediste1/btrbck/cli/CliMain.java

```diff
@@ -125,35 +125,49 @@ void processCommand(String... args) throws IOException {
 			log.debug("Arguments: " + arguments);
 
 			String command = arguments.get(0);
-			if ("snapshot".equals(command)) {
-				cmdSnapshot();
-			} else if ("list".equals(command)) {
-				cmdList();
-			} else if ("push".equals(command)) {
-				cmdPush();
-			} else if ("pull".equals(command)) {
-				cmdPull();
-			} else if ("process".equals(command)) {
-				cmdProcess();
-			} else if ("prune".equals(command)) {
-				cmdPrune();
-			} else if ("create".equals(command)) {
-				cmdCreate();
-			} else if ("delete".equals(command)) {
-				cmdDelete();
-			} else if ("restore".equals(command)) {
-				cmdRestore();
-			} else if ("receiveSnapshots".equals(command)) {
-				cmdReceiveSnapshots();
-			} else if ("sendSnapshots".equals(command)) {
-				cmdSendSnapshots();
-			} else if ("lock".equals(command)) {
-				cmdLock();
-			} else if ("version".equals(command)) {
-				cmdVersion();
-			} else {
-				throw new DisplayException("Unknown command " + command);
-			}
+			if (null != command) switch (command) {
+        case "snapshot":
+          cmdSnapshot();
+          break;
+        case "list":
+          cmdList();
+          break;
+        case "push":
```

Comment:
Using tabs instead of spaces gives every kind of formatting problems, like losing them while cut&pasting from console or browsers, wrong formatting on ides with different tab settings, etc.
That's why every project out there uses spaces!

Also, changing the indenting size is just a matter of pressing ctrl-shift-f on an IDE like netbeans: no problem at all reformatting the whole source set to a different settings.

This said, if you really prefer tabs I can configure my IDE (netbeans) to use them: the project is yours and I'll gladly adapt myself to your standards :)